buildpipeline/pipelinejobs.groovy

```diff
@@ -0,0 +1,55 @@
+// Import the utility functionality.
+
+import jobs.generation.JobReport;
+import jobs.generation.Utilities;
+import org.dotnet.ci.pipelines.Pipeline
+
+// The input project name (e.g. dotnet/core-setup)
+def project = GithubProject
+// The input branch name (e.g. master)
+def branch = GithubBranchName
+
+// **************************
+// Define innerloop testing. Any configuration in ForPR will run for every PR but all other configurations
+// will have a trigger that can be
+// **************************
+
+def alpine36Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/alpine.3.6.groovy')
+
+def configurations = [
+    ['TGroup':"netcoreapp", 'Pipeline':alpine36Pipeline, 'Name':'Alpine.3.6' ,'ForPR':"", 'Arch':['x64']],
+]
+
+configurations.each { config ->
+ ['Debug', 'Release'].each { configurationGroup ->
+  (config.Arch ?: ['x64', 'x86']).each { archGroup ->
+    def triggerName = "${config.Name} ${archGroup} ${configurationGroup} Build"
+
+    def pipeline = config.Pipeline
+    def params = ['TGroup':config.TGroup,
+                  'CGroup':configurationGroup,
+                  'AGroup':archGroup,
+                  'TestOuter': false]
+
+    // Add default PR triggers for particular configurations but manual triggers for all
+    if (config.ForPR.contains("${configurationGroup}-${archGroup}")) {
+        pipeline.triggerPipelineOnEveryGithubPR(triggerName, params)
+    }
+    else {
+        pipeline.triggerPipelineOnGithubPRComment(triggerName, params)
+    }
+
+    // Add trigger for all configurations to run on merge
+    pipeline.triggerPipelineOnGithubPush(params)
+
+    // Add optional PR trigger for Outerloop test runs
+    params.TestOuter = true
+    pipeline.triggerPipelineOnGithubPRComment("Outerloop ${triggerName}", params)
```

Comment:
without testing, what's the purpose of adding an outerloop leg?

Reply:
Good point, that was an artefact of copy and paste from corefx version of this file. The same actually applies to the coreclr version of this file that I've merged in yesterday.